release-notes/opensearch-release-notes-1.3.7.md

```diff
@@ -0,0 +1,75 @@
+* Upgraded loader-utils, async, cross-fetch, terser, node-fetch, minimatch, moment, jsdom, qs, execa to resolve CVE ([#555](https://github.com/opensearch-project/dashboards-reports/pull/555))
+* Upgraded detekt, snakyaml, and ktlint to resolve CVE ([#557](https://github.com/opensearch-project/dashboards-reports/pull/557))
+* Upgraded decode-uri-component to resolve CVE-2022-38900 ([#559](https://github.com/opensearch-project/dashboards-reports/pull/559))
+
+### OpenSearch Cross Cluster Replication
+* Include default index settings during leader setting validation ([#609](https://github.com/opensearch-project/cross-cluster-replication/pull/609))
+
+### OpenSearch Dashboards Visualizations
+* Bump node library versions ([#133](https://github.com/opensearch-project/dashboards-visualizations/pull/133))
+* Force ansi-regex to ^5.0.1 ([#135](https://github.com/opensearch-project/dashboards-visualizations/pull/135))
+
+## INFRASTRUCTURE
+
+### OpenSearch k-NN
+* Add Windows Build.sh Related Changes ([#602](https://github.com/opensearch-project/k-NN/pull/602))
+* Apply Spotless ([#640](https://github.com/opensearch-project/k-NN/pull/640))
+* Increment version to 1.3.7 ([#641](https://github.com/opensearch-project/k-NN/pull/641))
+* Get rid of Rachet to fix failing build ([#648](https://github.com/opensearch-project/k-NN/pull/648))
+* Exclude jacocoTestReport in distribution build script ([#652](https://github.com/opensearch-project/k-NN/pull/652))
+* Update build script to build JNI lib and gradlew assemble ([#659](https://github.com/opensearch-project/k-NN/pull/659))
+
+### OpenSearch Ml Commons
+* windows ci build ([#595](https://github.com/opensearch-project/ml-commons/pull/595))
+
+### OpenSearch Dashboards Reporting
+* Enable windows build ([#562](https://github.com/opensearch-project/dashboards-reports/pull/562))
+
+### OpenSearch Cross Cluster Replication
+* Update the CI workflow to run the integ tests on all platforms ([#649](https://github.com/opensearch-project/cross-cluster-replication/pull/649))
+
+### OpenSearch Dashboards Visualizations
+* Add support for windows ([#139](https://github.com/opensearch-project/dashboards-visualizations/pull/139))
+
+## MAINTENANCE
+
+### OpenSearch Ml Commons
+* Increment version to 1.3.7-SNAPSHOT ([#461](https://github.com/opensearch-project/ml-commons/pull/461))
+* backport: Address CVE-2022-42889 by updating commons-text ([#520](https://github.com/opensearch-project/ml-commons/pull/520))
+* force protobuf-java version as 3.21.9 ([#587](https://github.com/opensearch-project/ml-commons/pull/587))
+* fix junit version ([#596](https://github.com/opensearch-project/ml-commons/pull/596))
+
+### OpenSearch Dashboards Reporting
+* Bump version to 1.3.7 ([#502](https://github.com/opensearch-project/dashboards-reporting/pull/502))
+
+### OpenSearch Dashboards Visualizations
+* Version bump to 1.3.7 ([#131](https://github.com/opensearch-project/dashboards-visualizations/pull/131))
+
+## SECURITY FIXES
```

Comment:
Can you fix the category according to the template here: https://github.com/opensearch-project/opensearch-plugins/blob/f0fb80e46416bf13ceb991b9eac1c8e82eefb247/RELEASE_NOTES.md

Reply:
The category is present in cross-cluster-replication release notes https://github.com/opensearch-project/cross-cluster-replication/blob/1.3/release-notes/opensearch-cross-cluster-replication.release-notes-1.3.7.0.md. Should I move it under bug-fixes? @gaiksaya

Reply:
Yes they need to fix it too. Not sure if it should be bug fix or maintenance. Maybe check others' release notes to get an idea